commons/zenoh-config/src/lib.rs

```diff
@@ -1021,6 +1021,7 @@ fn load_external_plugin_config(title: &str, value: &mut Value) -> ZResult<()> {
 #[derive(Debug, Clone)]
 pub struct PluginLoad {
     pub name: String,
```

Comment:
In fact, `name` and `paths`have the same purpose: it's a information for loader where to look for corresponding plugin DLL. Probably the `name` should near the `paths` and be optional also